src/docs/sphinx/dev_guide/component_org.rst

```diff
@@ -139,14 +139,14 @@ Top-level component directory
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 The top-level component directory contains a `CMakeLists.txt`, e.g., 
-`axom/src/components/sidre/CmakeLists.txt`, which contains the following items:
+`axom/src/components/sidre/CMakeLists.txt`, which contains the following items:
 
   #. Checks for necessary dependencies with useful error or warning messages; 
```

Comment:
Is the remark about useful error or warning message still relevant here? If not, remove mention of it in text. If so, include in the example.